media/plg_system_tour/js/tours.js

```diff
@@ -0,0 +1,55 @@
+(function ($) {
+    $(document).ready(function() {
```

Comment:
@Hutchy68 then we **have** to find another one without jQuery. One of the major goals for Joomla 4 is to be jQuery free, so please lets not break this goal here